next/components/forms/Groups/InputSelectGroup.tsx

```diff
@@ -0,0 +1,148 @@
+import { EnumOptionsType } from '@rjsf/utils'
```

Comment:
you'll likely want a way of removing rows in this component as well - check around the team, it's possible someone's been looking into this already, and contact Patrik in the channel for how it should look if it does not exist yet